src/engine/rendergl.cpp

```diff
@@ -1854,7 +1854,8 @@ void drawfadedslice(float start, float length, float x, float y, float size, flo
 }
 
 float cursorx = 0.5f, cursory = 0.5f;
-vec cursordir(0, 0, 0);
+float crosshairx = 0.5f, crosshairy = 0.5f;
```

Comment:
why not use `vec2 crosshair_pos `instead of two floats?

Reply:
Because I wanted to make it similar to cursorx and cursory to make the code more consistent.

Reply:
why not make `float cursorx` and `cursory` into a `vec2 cursor_pos` ?

Reply:
Because that was not part of what I changed. You have to limit it somewhere.

Reply:
ok c: